examples/real-world/containers/Root.dev.js

```diff
@@ -1,24 +1,20 @@
-import React, { Component, PropTypes } from 'react'
+import React, { PropTypes } from 'react'
 import { Provider } from 'react-redux'
 import routes from '../routes'
 import DevTools from './DevTools'
 import { Router } from 'react-router'
 
-export default class Root extends Component {
-  render() {
-    const { store, history } = this.props
-    return (
-      <Provider store={store}>
-        <div>
-          <Router history={history} routes={routes} />
-          <DevTools />
-        </div>
-      </Provider>
-    )
-  }
-}
+const Root = ({ store, history }) =>
```

Comment:
Is React able to parse the `displayName` from named arrow functions? If not it might be good to add a`displayName` property to the exported functions for stuff like `ReactPerf`.

Unless the new `ReactPerf` that @gaearon is working on handles that differently?